### PR TITLE
fix(binaries): add libssl section

### PR DESCRIPTION
**Pull request by steebchen**

### CODE REVIEW

File: binaries/Readme.md [hunk 1]
```diff
@@ -122,7 +123,18 @@ of OpenSSL versions. This results in a few binaries which work on a large select
...
 
 Initially, we built specific binaries for given platforms (e.g. Netlify), which resulted in many 
-binaries which couldn't be shared with similar platforms and it was hard to maintain. 
+binaries which couldn't be shared with similar platforms and it was hard to maintain.
+
+### OpenSSL
+
+Prisma uses [OpenSSL](https://www.openssl.org/) to enable a secure communication to remote databases.
+This means OpenSSL has to be either shipped with the binary ("static linking") or exist on
+the users's machine ("dynamic linking"). As it is critical to always run the latest OpenSSL due to security
```
Comment (janpio) [inline binaries/Readme.md]:
```suggestion
the users's machine ("dynamic linking"). 

As it is critical to always run the latest OpenSSL due to security
```

File: binaries/Readme.md [hunk 2]
```diff
@@ -122,7 +123,18 @@ of OpenSSL versions. This results in a few binaries which work on a large select
...
+reasons, Prisma dynamically links OpenSSL so the user can update OpenSSL (usually with their native operating
+system package manager) independent of Prisma. Installing OpenSSL downloads a few files in common directories,
+which are usually named after the package name `libssl`, such as `libssl.so.10`, and the Prisma binaries
+automatically find and use these files. If they they don't exist on the user's system or are incompatible, Prisma
```
Comment (sorenbs) [inline binaries/Readme.md]: Double they

> If they they don't

Comment (janpio) [inline binaries/Readme.md]:
```suggestion
automatically find and use these files. If they they do not exist on the user's system or are incompatible, Prisma
```

Comment (divyenduz) [inline binaries/Readme.md]: Internal: Trying inline suggestion

**Comment by janpio** [comment]
This is part of `master` already, so merging this as well.

**Pull request merged**